queries.sql

```diff
@@ -19,4 +21,54 @@ SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
 
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- query and update animals table -------------------------------------
+
+ALTER TABLE animals ADD COLUMN species varchar(50);
+
+BEGIN; -- starts transaction
+
+UPDATE animals SET species = 'unspecified';
+
+ROLLBACK; -- ends transaction
+
+BEGIN; -- starts transaction
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+UPDATE animals SET species = 'pokemon' WHERE species = '';
+
+COMMIT; -- ends transaction
+
+BEGIN; -- starts transaction
+
+DELETE FROM animals;
+
+ROLLBACK;
```

Comment:
- [ ] Good job deleting all records, brave move! However, it would be great to add a query to verify after the `ROLLBACK` that your data still exists:

![Captura de pantalla 2022-06-15 a la(s) 6 37 15 p  m](https://user-images.githubusercontent.com/83717285/173959731-fc6e2b44-4705-4284-8c9b-a51ac44d57ce.png)